src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/Rfc3161TimestampTokenInfo.cs

```diff
@@ -165,4 +165,4 @@ internal static byte[] CopyFromNative(ref Rfc3161TimestampWin32.CRYPTOAPI_BLOB b
         }
 #endif
     }
-}
\ No newline at end of file
+}
```

Comment:
You'll want to exclude this file from the PR, as well.

Reply:
Thanks! Fixed.